src/app/inventory/item-types.ts

```diff
@@ -161,14 +160,42 @@ export interface DimItem {
   metricObjective?: DestinyObjectiveProgress;
   /** Metrics that can be used with this item. */
   availableMetricCategoryNodeHashes?: number[];
+  /** for D2 Y3 armor, this is the type and capacity information */
+  energy: DestinyItemInstanceEnergy | null;
+  /** If this item is a masterwork, this will include information about its masterwork properties. */
+  masterworkInfo: DimMasterwork | null;
+  /** If this exists, it's the limit of an item's PL. If NOT, display no information. Maybe it's unlimited PL. Maybe it's a weird item. */
+  powerCap: number | null;
+  /** Some items have a "flavor objective", such as emblems that track stats. */
+  flavorObjective: DimFlavorObjective | null;
+  /** an item's current breaker type, if it has one */
+  breakerType: DestinyBreakerTypeDefinition | null;
+  /** Information about how this item works with infusion. */
+  infusionQuality: DestinyItemQualityBlockDefinition | null;
+  /** The DestinyVendorDefinition hash of the vendor that can preview the contents of this item, if there is one. */
+  previewVendor?: number;
+  /** Localized string for where this item comes from... or other stuff like it not being recoverable from collections */
+  displaySource?: string;
+  /** The state of this item in the user's D2 Collection */
+  collectibleState?: DestinyCollectibleState;
+  collectibleHash?: number;
+  /** The DestinyCollectibleDefinition sourceHash for a specific item (D2). */
+  source?: number;
+  /** Extra pursuit info, if this item is a quest or bounty. */
+  pursuit: DimPursuit | null;
+  /** Information about this item as a plug. Mostly useful for mod collectibles. */
+  plug?: {
+    energyCost: number;
+    costElementIcon: string;
+  };
 
   /** "Touch" the item to mark it as having been manually moved. */
   updateManualMoveTimestamp(): void;
 
   /** Check if this item is from D1. Inside an if statement, this item will be narrowed to type D1Item. */
   isDestiny1(): this is D1Item;
-  /** Check if this item is from D2. Inside an if statement, this item will be narrowed to type D2Item. */
-  isDestiny2(): this is D2Item;
+  /** Check if this item is from D2. */
+  isDestiny2(): boolean;
```

Comment:
is it still being used in some places?

Reply:
Yup - there are places where we still want to do something only for D2, and aren't using it for type info.